ccpp/data/GFS_typedefs.F90

```diff
@@ -2554,6 +2555,35 @@ subroutine coupling_create (Coupling, IM, Model)
 !     Coupling%zorlwav_cpl  = clear_val
 !   endif
 
+    if (Model%cplflx .or. Model%cpllnd) then
+      allocate (Coupling%dlwsfc_cpl (IM))
+      Coupling%dlwsfc_cpl = clear_val
+      allocate (Coupling%dswsfc_cpl (IM))
+      Coupling%dswsfc_cpl = clear_val
+      allocate (Coupling%psurfi_cpl (IM))
+      Coupling%psurfi_cpl = clear_val
+      allocate (Coupling%nswsfc_cpl (IM))
```

Comment:
Also, please double-check that the `active` attribute is updated for ALL variables in GFS_typedefs.meta. It looks to me like some variables didn't get `.or. flag_for_land_coupling` added to reflect the new conditions of allocation in the actual Fortran.

Reply:
This is fixed with [b96b84c](https://github.com/NOAA-EMC/fv3atm/pull/591/commits/b96b84c51a86a388d5285e802f224ec2c0266ec1). Thanks for pointing those issues.